src/App.css

```diff
@@ -1,38 +1,23 @@
-.App {
+.container{
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.image-grid{
+  display: grid;
+  margin: 10px;
+/*  border: 1px solid red;*/
+  grid-gap: 10px;
```

Comment:
Please use gap here, as grid-gap is deprecated now. I used this during workshop1 by mistake